### DISCUSSION TITLE
About SAM-LLaVA

### DISCUSSION THREAD

**Discussion by youngwanLEE** [PixArt-alpha/PixArt-alpha#11] [Q&A]
Hi, I'm really impressed by your great work.

I want to use your SAM-LLaVA caption dataset but failed to find where it is.

Could you let me know the location?

Thanks in advance.

**Comment by lawrence-cj** [comment]
We are still working on it and will release in the near future. Plz stay tuned.

**Reply by youngwanLEE** [reply]
Thanks for your quick reply :)
